deployments/deploy-ssh.sh

```diff
@@ -672,6 +674,10 @@ spec:
       value: "${NFS_PATH}"
     - name: SKIP_PX_OPERATOR_UPGRADE
       value: "${SKIP_PX_OPERATOR_UPGRADE}"
+    - name: CLOUD_CRED_CONFIG
+      value: "${CLOUD_CRED_CONFIG}"
+    - name: CONFIG_JSON
+      value: "${CONFIG_JSON}"
```

Comment:
Please remove all this. I think we discussed about this.

Reply:
done